### PR TITLE
Put Update My Machine Modal button to stack template section

### PR DESCRIPTION
**Pull request by hakankaradis**
We will show stack update notification button next to each stack title if stack needs to be updated.
## Description

When user click the notification button we will show stack update modal.
## Motivation and Context
#8645
## How Has This Been Tested?

WIP
## Screenshots (if appropriate):

http://recordit.co/joaDw8IFSP
http://recordit.co/eJ7ESbG1NM
## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
## Checklist:
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by sinan** [comment]
thx @hakankaradis you need one more pass here, with some help from @ftasdelen

File: client/home/lib/stacks/components/stacktemplateitem/index.coffee [hunk 1]
```diff
@@ -2,9 +2,43 @@ kd = require 'kd'
...
+
+    super
+
+    @state        =
+      coordinates :
+        left      : 0
+        top       : 0
+      showWidget  : no
+
+    $('.kdscrollview').on 'scroll', @bound 'scrollOnPage'
```
Comment (sinan) [inline client/home/lib/stacks/components/stacktemplateitem/index.coffee]: use `_.debounce` here

**Comment by hakankaradis** [comment]
@sinan 
here is final screencast http://recordit.co/4rW8bJ7rPP

**Comment by hakankaradis** [comment]
http://recordit.co/Tl1qvmUiOK
@sinan

**Comment by sinan** [comment]
👍

**Pull request merged**